generate_data.py

```diff
@@ -268,7 +269,8 @@ def positive_int(num):
 parser.add_argument('--encoded_file_dir', default=None, type=writable_dir)
 parser.add_argument('--frame_offset', default=0, type=positive_int)
 parser.add_argument('--num_frames', default=-1, type=positive_int)
-
+parser.add_argument('--enable-vmaf', dest='vmaf', action='store_true')
```

Comment:
Can you sort these in alphabetic order and add default=False instead of parser.set_defaults?